src/shared/getFunction.js

```diff
@@ -0,0 +1,10 @@
+export let functions = {};
+
+export default function getFunction ( str, i ) {
+	if ( functions[ str ] ) return functions[ str ];
+
+	let args = new Array( i );
+	while ( i-- ) args[i] = `_${i}`;
+
+	return ( functions[ str ] = new Function( args.join( ',' ), `return (${str})` ) );
```

Comment:
I think @MartinKolarik has it covered, we just need to (re)move this last line, or move it into the parser. Trying it out now...